core/src/test/scala/org/apache/spark/scheduler/WorkerDecommissionSuite.scala

```diff
@@ -76,7 +76,9 @@ class WorkerDecommissionSuite extends SparkFunSuite with LocalSparkContext {
     // decom.sh message passing is tested manually.
     val sched = sc.schedulerBackend.asInstanceOf[StandaloneSchedulerBackend]
     val execs = sched.getExecutorIds()
-    execs.foreach(execId => sched.decommissionExecutor(execId, ExecutorDecommissionInfo("", false)))
+    // Make the executors decommission, finish, exit, and not be replaced.
+    val execsAndDecomInfo = execs.map((_, ExecutorDecommissionInfo("", false)))
+    sched.decommissionExecutors(execsAndDecomInfo, adjustTargetNumExecutors = true)
```

Comment:
I am confused here: `adjustTargetNumExecutors = true` means that the executor should be replaced (IIUC). Whereas the comment above says "not be replaced".

Reply:
Nope, please re-read the code. This matches the killImplemwntation.

Reply:
^^^ @holdenk ... any thoughts/followup on this ?

Reply:
My reasoning here is that by adjusting the executors down here we ensure the task isn't being re-launched and a re-launched executor so it improves the test.

Reply:
Okay sure :-) But I guess that dynamic allocation isn't even enabled for this test so executors shouldn't just launch again on this node -- unless a new worker appears (this is standalone local cluster testing) out of nowhere. Spark only launches executors when it detects spare capacity for that -- all that mesos based resourceOffers paradigm. 

In any case, I understand your intent here now and I am totally cool with that. So please consider this resolved.